apps/patient/src/utils/general.ts

```diff
@@ -7,11 +7,9 @@ import timezone from 'dayjs/plugin/timezone';
 import utc from 'dayjs/plugin/utc';
 import costcoLogo from '../assets/costco_small.png';
 import walgreensLogo from '../assets/walgreens_small.png';
-import walmartLogo from '../assets/walmart_small.svg';
```

Comment:
Do we want to keep the Walmart logo? I get that price didnt make a difference but I feel like it's nice to be able to see the logo?

Reply:
was thinking the same thing, but maybe this is something we don't give away for free to a pharmacy? But it is a nicer looking interface

Reply:
Yeah that makes sense to me. Idk